diff --git a/ryu/ofproto/ofproto_common.py b/ryu/ofproto/ofproto_common.py
index ffdf1a4..4b29005 100644
--- a/ryu/ofproto/ofproto_common.py
+++ b/ryu/ofproto/ofproto_common.py
@@ -24,8 +24,8 @@ assert calcsize(OFP_HEADER_PACK_STR) == OFP_HEADER_SIZE
 # Note: IANA assigned port number for OpenFlow is 6653
 # from OpenFlow 1.3.3 (EXT-133).
 # Some applications may still use 6633 as the de facto standard though.
-OFP_TCP_PORT = 6653
-OFP_SSL_PORT = 6653
+OFP_TCP_PORT = 10002
+OFP_SSL_PORT = 10002
 OFP_TCP_PORT_OLD = 6633
 OFP_SSL_PORT_OLD = 6633

